packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -329,7 +329,7 @@ class AnimationController extends Animation<double>
   /// [AnimationController.unbounded] constructor.
   final AnimationBehavior animationBehavior;
 
-  /// Returns an [Animation<double>] for this animation controller, so that a
+  /// Returns an `Animation<double>` for this animation controller, so that a
```

Comment:
Hm, there's kind of a regression here. Previously, this would correctly cross-link to the docs for `Animation`. Now, it will not link anymore. Looks like the `[Animation<double>]` also rendered correctly:
<img width="386" alt="Screenshot 2024-04-11 at 4 54 27 PM" src="https://github.com/flutter/flutter/assets/1227763/dddd6aa0-5a18-4f1b-bbd7-6b142855a783">

Should/could the lint maybe not flag these cases?

Reply:
Ahhhhh bummer. This might be a case where dartdoc handles more advanced syntax than the analyzer (currently).

>Should/could the lint maybe not flag these cases?

By "these cases," do you mean angle brackets in square brackets? That's a possible workaround.

Reply:
> By "these cases," do you mean angle brackets in square brackets?

Yes, kind of. I mean cases where angle brackets are used in square brackets to mark generics. These should probably still be illegal: `[><]`, `[Animation<<double>>]`, etc.

> This might be a case where dartdoc handles more advanced syntax than the analyzer

What's the practical implication of this analyzer vs. dartdoc mismatch?